keras/distribution/distribution_lib.py

```diff
@@ -14,13 +14,11 @@
 
 import numpy as np
 
-# TF is used for dataset sharding.
-import tensorflow as tf
-
 from keras.api_export import keras_export
 from keras.backend import KerasTensor
 from keras.backend import distribution_lib
 from keras.backend.common import global_state
+from keras.utils.module_utils import tensorflow as tf
```

Comment:
Thanks for the fix. Can we move this to the place that tf is used (eg the distribute_dataset method.)